Description: Initscript creates RUN_DIR , make sure it's owned by couchdb
 Use install to make COUCHDB_USER own the RUN_DIR being created.
Author: Laszlo Boszormenyi (GCS) <gcs@debian.hu>
Bug-Debian: http://bugs.debian.org/681549
Last-Update: 2012-11-18

---

--- couchdb-1.2.0.orig/etc/init/couchdb.tpl.in
+++ couchdb-1.2.0/etc/init/couchdb.tpl.in
@@ -83,7 +83,8 @@ run_command () {
 start_couchdb () {
     # Start Apache CouchDB as a background process.
 
-    mkdir -p "$RUN_DIR"
+    test -e "$RUN_DIR" || \
+        install -m 755 -o "$COUCHDB_USER" -g "$COUCHDB_USER" -d "$RUN_DIR"
     command="$COUCHDB -b"
     if test -n "$COUCHDB_STDOUT_FILE"; then
         command="$command -o $COUCHDB_STDOUT_FILE"
